### PR TITLE
pagination added for favorites

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

**Comment by mjoyce91** [comment]
Let's make sure to merge at the same time as https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/740

**Comment by mjoyce91** [comment]
@elizabeth-jimenez - this conditional filter becomes a little weird with the update. Maybe "Filter by" should become the later filter since it's displayed conditionally. Or it could be grayed out until a season is chosen.

File: talentmap_api/available_positions/views/available_position.py [hunk 1]
```diff
@@ -36,16 +36,27 @@ class AvailablePositionFavoriteListView(APIView):
 
     permission_classes = (IsAuthenticated,)
 
+    schema = AutoSchema(
+        manual_fields=[
+            coreapi.Field("page", location='query', type='integer', description='Page size'),
```
Comment (scott062) [inline talentmap_api/available_positions/views/available_position.py]: Let change this to page index

File: talentmap_api/projected_vacancies/views/projected_vacancy.py [hunk 2]
```diff
@@ -19,16 +23,27 @@ class ProjectedVacancyFavoriteListView(APIView):
 
     permission_classes = (IsAuthenticated,)
 
+    schema = AutoSchema(
+        manual_fields=[
+            coreapi.Field("page", location='query', type='integer', description='Page size'),
```
Comment (scott062) [inline talentmap_api/projected_vacancies/views/projected_vacancy.py]: Lets change this to page index as well

**Pull request merged**